openedx/core/djangoapps/user_api/accounts/settings_views.py

```diff
@@ -72,6 +74,12 @@ def account_settings(request):
         return redirect(url)
 
     context = account_settings_context(request)
+
+    try:
+        context = AccountSettingsRenderStarted().run_filter(context=context)
+    except AccountSettingsRenderStarted.ErrorFilteringContext as exc:
```

Comment:
I was about to test this in conjunction the other PR ([openedx-filters PR 46](https://github.com/openedx/openedx-filters/pull/46)) but then realized this is not going to work. ErrorFilteringContext is not defined as one of the filter exceptions. Also, try to follow [this](https://github.com/openedx/edx-platform/blob/f3149f55f09b8c6377eaa2c7e5c801ce657a7db2/lms/djangoapps/certificates/views/webview.py#L646) implementation.